frontend/packages/lib-components/src/atoms/buttons/InlineButton.tsx

```diff
@@ -45,6 +45,7 @@ const StyledButton = styled.button`
 
   svg {
     margin-right: ${defaultMargins.xs};
+    font-size: 20px;
```

Comment:
This should probably be a relative unit as well so that the icon grows with text size.